QSB/TransformSync/TransformSync.cs

```diff
@@ -108,6 +111,11 @@ private void Update()
 
                     SyncedTransform.localPosition = Vector3.SmoothDamp(SyncedTransform.localPosition, transform.position, ref _positionSmoothVelocity, SmoothTime);
                     SyncedTransform.localRotation = QuaternionHelper.SmoothDamp(SyncedTransform.localRotation, transform.rotation, ref _rotationSmoothVelocity, Time.deltaTime);
+
+                    if (Override())
```

Comment:
Ah, there are two of these. Hmmm. I suggest we find a way to properly teleport the probe. Scratch the above comment.